examples/v1alpha2/http-routing/gateway.yaml

```diff
@@ -0,0 +1,17 @@
+kind: Gateway
+apiVersion: gateway.networking.k8s.io/v1alpha2
+metadata:
+  name: prod-web
+spec:
+  gatewayClassName: acme-lb
+  listeners:
+  - protocol: HTTP
+    port: 80
+    routes:
+      namespaces:
+        from: All
+        selector:
+          matchLabels:
+            gateway: prod-web-gw
+      kinds:
+      - kind: HTTPRoute
```

Comment:
None of this is necessary

Reply:
This is now removed with adcf37a71d913be2cb3d14f9d269501346e7ca25, and the follow up is c03ae32d6f1e5dc105e7413f459c4d5dd80c5877